app/src/main/java/org/jellyfin/mobile/webapp/RemoteVolumeProvider.kt

```diff
@@ -0,0 +1,25 @@
+package org.jellyfin.mobile.webapp
+
+import android.media.AudioManager
+import android.media.VolumeProvider
+import org.jellyfin.mobile.utils.Constants
+
+class RemoteVolumeProvider(private val webappFunctionChannel: WebappFunctionChannel) : VolumeProvider(VOLUME_CONTROL_ABSOLUTE, 100, 0) {
```

Comment:
```suggestion
class RemoteVolumeProvider(
    private val webappFunctionChannel: WebappFunctionChannel
) : VolumeProvider(VOLUME_CONTROL_ABSOLUTE, 100, 0) {
```

I prefer multi-line constructors if they become too long